### PR TITLE
Translate "Hello World" Page to Tagalog

### LINKED ISSUE #2: Basics: Hello World

### PR DESCRIPTION
**Pull request by jorenrui**
# Story Title

Translate "Hello World" Page to Tagalog

## Background

I've translated the React docs's [Hello World](https://reactjs.org/docs/hello-world.html) page to Tagalog. I've left some of the word in english like `components` and `library`.

This is my first time translating docs so I'm not quite sure if my words are correct. Feel free to make some suggestions.

## Linked Issues

Resolves #2

### CODE REVIEW

File: content/docs/hello-world.md [hunk 1]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Ipinapakita rito ang isang heading na nagsasabing "Hello, world!" sa pahina.
```
Comment (elrmndo) [inline content/docs/hello-world.md:18]: We should proceed making this "Kamusta, mundo!", what you think? 😉

Comment (jorenrui) [reply content/docs/hello-world.md:18]: Ow I didn't translate "Hello, world" cause it is linked to a codepen 😅 which says 

image

image

So I left it at that cause I can't edit the CodePen.

Moreover, when I checked the other React docs translation like Spanish.

image

They left it at Hello World too.

**Review by elrmndo — APPROVED** [review]

File: content/docs/hello-world.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 id: hello-world
-title: Hello World
+title: Kamusta Mundo
 permalink: docs/hello-world.html
 prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+Isang simpleng halimbawa ng React ay ito:
```
Comment (elrmndo) [inline content/docs/hello-world.md:9]: We can drop "ay ito" and make it as "Isang simpleng halimbawa ng React:"

File: content/docs/hello-world.md [hunk 3]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
...
-
-## Let's Get Started! {#lets-get-started}
-
-Keep scrolling down, and you'll find the link to the [next chapter of this guide](/docs/introducing-jsx.html) right before the website footer.
+>Ang gabay na ito ay madalas gumagamit ng mga panibagong JavaScript syntax sa mga halimbawa. Kung hindi ka na gumagamit ng JavaScript sa nakalipas na mga taon, [
+ang tatlong puntong ito](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) ay makakatulong para mairaos ka sa mga susunod na aralin.
 
+## Halina't Magsimula! {#lets-get-started}
 
+Patuloy na mag-scroll pababa at iyong makikita ang mga link sa mga [susunod na kabanata](/docs/introducing-jsx.html) bago mismo ang footer ng website.
```
Comment (elrmndo) [inline content/docs/hello-world.md:47]: "at iyong makikita ang link sa susunod na kabanata bago ang footer ng website."

File: content/docs/hello-world.md [hunk 4]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
...
->Note
+>Tandaan
 >
->This guide occasionally uses some of the newer JavaScript syntax in the examples. If you haven't worked with JavaScript in the last few years, [these three points](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) should get you most of the way.
-
-
-## Let's Get Started! {#lets-get-started}
-
-Keep scrolling down, and you'll find the link to the [next chapter of this guide](/docs/introducing-jsx.html) right before the website footer.
+>Ang gabay na ito ay madalas gumagamit ng mga panibagong JavaScript syntax sa mga halimbawa. Kung hindi ka na gumagamit ng JavaScript sa nakalipas na mga taon, [
```
Comment (elrmndo) [inline content/docs/hello-world.md:42]: Ang gabay na ito ay ~~madalas~~ gumagamit ng mga panibagong JavaScript syntax sa mga halimbawa. Kung hindi ka na gumagamit ng JavaScript sa nakalipas na mga taon,

File: content/docs/hello-world.md [hunk 5]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
...
+ang tatlong puntong ito](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) ay makakatulong para mairaos ka sa mga susunod na aralin.
```
Comment (elrmndo) [inline content/docs/hello-world.md:43]: ang tatlong puntong ito](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) ay makakatulong para ~~mairaos~~ makasunod ka sa mga susunod na aralin.

File: content/docs/hello-world.md [hunk 6]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
...
 
-React is a JavaScript library, and so we'll assume you have a basic understanding of the JavaScript language. **If you don't feel very confident, we recommend [going through a JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level** and enable you to follow along this guide without getting lost. It might take you between 30 minutes and an hour, but as a result you won't have to feel like you're learning both React and JavaScript at the same time.
+Ang React ay isang JavaScript library kung kaya't ina-assume namin na may sapat kang kaalaman sa lengguwaheng JavaScript. **Kung may agam-agam at pagdududa sa iyong sarili, inirerekumenda namin na [sumubok munang mag-aral ng isang JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para masuri ang antas ng iyong kaalaman** at makasunod sa gabay na ito nang hindi naguguluhan. Maari kang abutin ng mula sa 30 minuto hanggang sa isang oras, ngunit bilang resulta hindi mo mararamdaman na nag-aaral ka ng React at JavaScript nang magkasabay.
```
Comment (elrmndo) [inline content/docs/hello-world.md:38]: Ang React ay isang JavaScript library kung kaya't ~~ina-assume~~ aming ina-asume ~~namin~~ na may sapat kang kaalaman sa ~~lengguwaheng~~ JavaScript. **Kung may agam-agam at pagdududa ~~sa iyong sarili~~ ka sa iyong nalalaman, inirerekumenda namin na [sumubok munang mag-aral ng isang JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para masuri ang antas ng iyong kaalaman** at **nang** makasunod sa gabay na ito nang hindi naguguluhan. Maari kang abutin ng mula sa 30 minuto hanggang sa isang oras, ngunit bilang resulta hindi mo mararamdaman na nag-aaral ka ng React at JavaScript nang ~~magkasabay~~ sabay.

File: content/docs/hello-world.md [hunk 7]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
...
+Ito ang unang kabanata sa bai-baitang na gabay patungkol sa pinakamahalagang mga konsepto ng React. Ang mga listahan ng mga kabanata ay makikita sa navigation sidebar. Kung ikaw ay nagbabasa nito gamit ang iyong mobile device, ma-aaccess mo ang navigation sa pamamagitan ng pagpindot sa pindutan sa kanang sulok sa ibaba ng iyong screen.
 
-Every chapter in this guide builds on the knowledge introduced in earlier chapters. **You can learn most of React by reading the “Main Concepts” guide chapters in the order they appear in the sidebar.** For example, [“Introducing JSX”](/docs/introducing-jsx.html) is the next chapter after this one.
+Ang bawat kabanata sa gabay na ito ay karagdagang aral kasabay sa naunang mga kabanata. **Maari mong malaman ang kabuuan ng React sa pamamagitan ng pagbasa sa mga kabanata ng "Main Concepts" ayon sa pagkakasunod-sunod nito sa sidebar.** Halimbawa, [“Introducing JSX”](/docs/introducing-jsx.html) ang kasunod na kabanata pagkatapos nito.
```
Comment (elrmndo) [inline content/docs/hello-world.md:34]:
```suggestion
Ang bawat kabanata sa gabay na ito ay nagdadagdag ng kaalaman ukol sa mga naunang mga kabanata. **Maari mong malaman ang kabuuan ng React sa pamamagitan ng pagbasa sa mga kabanata ng "Main Concepts" ayon sa pagkakasunod-sunod nito sa sidebar.** Halimbawa, [“Introducing JSX”](/docs/introducing-jsx.html) ang kasunod na kabanata pagkatapos nito.
```

File: content/docs/hello-world.md [hunk 8]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
...
 
-## How to Read This Guide {#how-to-read-this-guide}
-
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+Dito sa gabay, pag-aaralan natin ang mga pundasyon ng React apps: elements at components. Kapag naging dalubhasa ka na sa mga ito, maaari ka nang gumawa ng mga komplikadong apps gamit ang mga maliliit na reusable na mga piraso.
 
 >Tip
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>Itong gabay ay dinesenyo para sa mga taong mas gusto **pag-aralan ang mga konsepto ng bai-baitang**. Kapag mas gusto mo matuto sa pamamagitan ng paggawa, pwede mong tignan ang aming [praktikal na tutorial](/tutorial/tutorial.html). Iyong mapapansin na ang gabay na ito at ang tutorial ay nagko-komplementaryo sa isa't isa.
```
Comment (elrmndo) [inline content/docs/hello-world.md:30]:
```suggestion
>Ang gabay na ito ay ginawa para sa mga taong mas gusto **pag-aralan ang mga konsepto ng isa-isa**. Kung mas gusto mong matuto sa pamamagitan ng paggawa, pwede mong tignan ang aming [praktikal na tutorial](/tutorial/tutorial.html). Iyong mapapansin na ang gabay na ito at ang tutorial ay komplementaryo sa isa't isa.
```

File: content/docs/hello-world.md [hunk 9]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
...
 
-This is the first chapter in a step-by-step guide about main React concepts. You can find a list of all its chapters in the navigation sidebar. If you're reading this from a mobile device, you can access the navigation by pressing the button in the bottom right corner of your screen.
+Ito ang unang kabanata sa bai-baitang na gabay patungkol sa pinakamahalagang mga konsepto ng React. Ang mga listahan ng mga kabanata ay makikita sa navigation sidebar. Kung ikaw ay nagbabasa nito gamit ang iyong mobile device, ma-aaccess mo ang navigation sa pamamagitan ng pagpindot sa pindutan sa kanang sulok sa ibaba ng iyong screen.
```
Comment (elrmndo) [inline content/docs/hello-world.md:32]:
```suggestion
Ito ang unang kabanata patungkol sa pinakamahalagang mga konsepto ng React. Ang mga listahan ng mga kabanata ay makikita sa navigation sidebar. Kung ikaw ay nagbabasa nito gamit ang iyong mobile device, ma-aaccess mo ang navigation sa pamamagitan ng pagpindot sa pindutan sa kanang ibaba ng iyong screen.
```

File: content/docs/hello-world.md [hunk 10]
```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
...
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Pindutin ang link na nasa itaas para magbukas ng isang online editor. Huwag mag-atubiling gumawa ng ilang mga pagbabago at tignan kung papaano ito nakakaapekto sa resulta. Karamihan ng mga pahina sa gabay ay mayroong mga halimbawang pwedeng i-edit katulad nito.
 
+## Papaano Basahin ang Gabay na ito {#how-to-read-this-guide}
 
-## How to Read This Guide {#how-to-read-this-guide}
-
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+Dito sa gabay, pag-aaralan natin ang mga pundasyon ng React apps: elements at components. Kapag naging dalubhasa ka na sa mga ito, maaari ka nang gumawa ng mga komplikadong apps gamit ang mga maliliit na reusable na mga piraso.
```
Comment (elrmndo) [inline content/docs/hello-world.md:26]:
```suggestion
Sa gabay na ito, pag-aaralan natin ang mga pundasyon ng mga React apps: elements at components. Kapag naging dalubhasa ka na sa mga ito, maaari ka nang gumawa ng mga komplikadong apps gamit ang mga maliliit na reusable na mga piraso o components.
```

File: content/docs/hello-world.md (hunk 1 above)
Comment (elrmndo) [inline content/docs/hello-world.md:18]:
```suggestion
Sa pahinang ito ipinapakita ang isang heading na nagsasabing "Hello, world!"
```

File: content/docs/hello-world.md [hunk 11]
```diff
@@ -1,12 +1,12 @@
 ---
 id: hello-world
-title: Hello World
+title: Kamusta Mundo
```
Comment (elrmndo) [inline content/docs/hello-world.md:3]:
```suggestion
title: Hello World
```

I think we retain ```Hello World``` just to be consistent with the codepen snippet

**Review by elrmndo — CHANGES_REQUESTED** [review]
I have reviewed the pr and have suggested some changes, feel free to let me know if you agree/disagree with the suggestion, stay safe!

File: content/docs/hello-world.md (hunk 11 above)
Comment (warrenca) [reply content/docs/hello-world.md:3]: @elrmndo Pwede ba nating palitan lahat ng `Hello, world!` ng `Kamusta, mundo!`? Was looking at other languages and binago nila yun in their language https://fr.reactjs.org/docs/introducing-jsx.html, https://pl.reactjs.org/docs/introducing-jsx.html, https://ru.reactjs.org/docs/introducing-jsx.html. It's just a nice touch imo.

File: content/docs/hello-world.md (hunk 6 above)
Comment (jorenrui) [reply content/docs/hello-world.md:38]: Amma change this `"naming [sumubukan mo munang magbasa"` to this instead `"naming [subukan mo munang magbasa"`

**Comment by jorenrui** [comment]
UPDATE: I've also translated the sidebar item and CodePen link to Tagalog

image
image

**Review by elrmndo — APPROVED** [review]
Cool! this this looks good to me.

**Pull request merged**